packages/vue-instantsearch/src/util/__tests__/renderCompat.test.js

```diff
@@ -0,0 +1,52 @@
+/**
+ * @jest-environment jsdom
+ */
+
+import { mount } from '../../../test/utils';
+import { renderCompat } from '../vue-compat';
+
+describe('renderCompat', () => {
+  it('should handle function components that return multiple children', () => {
+    const wrapper = mount({
+      render: renderCompat((h) => {
+        function Component({ text }) {
+          return h(
+            'div',
+            { attrs: { 'data-text': text } },
+            h('span', {}, ['1']),
+            h('span', {}, '2')
```

Comment:
I also added this test to make sure different ways of passing children work (rest args or single array)